source/quick-start.txt

```diff
@@ -7,193 +7,202 @@ Quick Start
+Create a MongoDB Cluster
+------------------------
 
-.. code-block:: sh
+Set up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  npm install --save dotenv
+After installing the Node MongoDB driver, create a MongoDB instance to store
+and manage your data. Complete the
+:atlas:`Get Started with Atlas <getting-started>` guide to set up a new
+Atlas account, free tier cluster (MongoDB instance), load datasets, and
+interact with the data.
 
-Next, create the ``.env`` file and add a variable called
-``MONGODB_URI``:
+After completing the steps in the Atlas guide, you should have a new MongoDB
+cluster deployed in Atlas, a new database user, and sample datasets loaded
+into your cluster.
 
-.. code-block:: sh
+Connect to your Cluster
+-----------------------
 
-  touch .env
-  echo "MONGODB_URI=<connection_uri>" >> .env
+In this step, we create and run an application that uses the Node.js MongoDB
+driver to connect to your instance of MongoDB and run a query on the sample
+data.
 
-Add your Atlas Connection String
---------------------------------
+To connect to Atlas using the Node.js driver, we provide instructions to the
+driver on where and how to connect to your instance in a string called the
+*connection string*.
 
-Open your newly created ``.env`` file. If you are unable to locate this
-file in your project directory, you may need to set your system to
-display hidden files. Once your ``.env`` is open, replace
-``<connection_uri>`` with your connection string from Atlas. Ensure you
-update ``<username>`` and ``<password>`` in your connection string.
+Log into your Atlas account and navigate to the **Clusters** section and
+click the **Connect** button for the cluster in which you want to connect
+as shown below.
 
-Connect to your Cluster
------------------------
+.. figure:: /includes/figures/atlas_connection_select_cluster.png
 
-For the final step, we'll write some code that tests your environment's
-connection to your instance of MongoDB.
+Follow the instructions in the dialog to whitelist your IP address and create
+a new MongoDB user with **atlasAdmin** permissions. Then proceed to
+**Choose a connection method**.
 
-Create a file in your project directory called ``index.js``:
+From the list of connection methods, select **Connect Your Application** and
+select the Node.js driver. Select the "Connection String Only" tab and click
+the **Copy** button to copy the *connection string* to your clipboard as
+shown below.
 
-.. code-block:: sh
+.. figure:: /includes/figures/atlas_connection_copy_string.png
 
-  touch index.js
+Save your connection string to a safe location that you can access in the
+next step.
 
-Open this file and paste in the following code:
+Next, create a file to contain your application called ``index.js`` in your
+``node_quickstart`` directory. Add the following code, replacing the ``uri``
+variable with your connection string. Make sure to replace the "<password>"
```

Comment:
I know Atlas pre-fills in the user for you in the connection string, just hoping there isn't much confusion here.

Reply:
Not sure if we're referring to the same thing, but I only mentioned that they need to replace "<password>" because that is not filled in by Atlas, probably for security reasons.

Reply:
Yeah, we're referring to the same thing.

Reply:
Ok, is there anything that you wanted to change then? The sentence mentions that they should replace the "`<password>`" placeholder and nothing else.

Reply:
I was just noting that we ask them to replace "`<password>`", which makes sense. I was wondering whether or not we need to make a note about the username, which I know does not need to be replaced because presumably Atlas auto-fills it in for users in the connection string. "Your username should appear in the connection string..." etc. This is a non-blocking comment and I am OK to go live as-is.

Reply:
I see. I don't think it's necessary since Atlas auto-fills it. If they're not using Atlas, they'll need to read some other literature on how to connect. Since there wasn't any link to a doc to describe how to connect without Atlas in the original doc, I'm guessing that was intentional.